building-with-ocean/asset-hosting.md

```diff
@@ -17,134 +17,136 @@ On Ocean Marketplace, a publisher must provide the link to the asset during publ
 
-#### Google Drive
+#### AWS
 
-Google Drive allows users to share files/folders with various access policies. Publishers must set the access policy such that anyone with the link can download the file when using Ocean Marketplace with Ocean Protocol's default [Provider](https://v4.provider.mumbai.oceanprotocol.com).
+AWS provides various options to host data and multiple configuration possibilities. Publishers are required to do their research and decide what would be the right choice. The below steps provide one of the possible ways to host data using AWS S3 bucket and publish it on Ocean Marketplace.
 
-**Step 1 - Get link**
+**Prerequisite**
 
-Open https://drive.google.com and upload the file you want to publish on the Ocean Marketplace. Right-click on the uploaded file and click the `Share` option. Set the file access policy correctly and click the `Copy link` button.
+Create an account on [AWS](https://aws.amazon.com/s3/). Users might also be asked to provide payment details and billing addresses that are out of this tutorial's scope.
 
-The file URL will be of the form `https://drive.google.com/file/d/<FILE-ID>/view?usp=sharing`, where the `<FILE-ID>` is the unique alphanumeric string. Verify if the URL is correct by entering it in a browser and check if the file is downloaded.
+**Step 1 - Create a storage account**
 
-![Google Drive link](../.gitbook/assets/publish-google-drive.png)
+**Go to AWS portal**
 
-**Step 2 - Create a downloadable link**
+Go to the AWS portal for S3: https://aws.amazon.com/s3/ and select from the upper right corner `Create an AWS account` as shown below.
 
-If you paste the copied URL into the browser, it will load an HTML page. Directly pasting the link on the publish page will publish the HTML page instead of a downloadable file URL. So, let's make a downloadable file URL.
+![Create an account - 1](../.gitbook/assets/aws-1.png)
 
-Note the `<FILE-ID>` from step 1 and create a URL as below.
+**Fill in the details**
 
-`https://drive.google.com/uc?export=download&id=<FILE-ID>`
+![Create an account - 2](../.gitbook/assets/aws-2.png))
 
-**Step 3 - Publish the asset using the generated link**
+**Create a bucket**
 
-After creating a downloadable file URL, fill the `File*` field with the downloadable URL created in step 2.
+After logging into the new account, search for the available services and select `S3` type of
+storage.
 
-![Publish - Google Drive file](../.gitbook/assets/publish-google-drive-2.png)
+![Create an account - 3](../.gitbook/assets/aws-3.png)
 
-_Note: Google Drive allows only shared files to be downloaded, as shown in the above steps. The above method does not work with the shared folder. As a workaround, publishers can upload a zip of a folder and upload it as a file._
+In order to create a S3 bucket, choose `Create bucket`.
 
-***
+![Create an account - 4](../.gitbook/assets/aws-4.png)
 
-#### Azure storage
```

Comment:
Why you have removed the Azure tutorial? 
If you closely look at the task, you'll see we need to remove onedrive and drive. Please bring it back